### ISSUE TITLE
[Usable-Command Palette]-Keyboard shortcuts are not included in the sub menu of New Tab list item.

### ISSUE THREAD

**Issue by DasaradhaKamjula** [microsoft/terminal#7912]
**User Experience:**
Shortcuts that are provided inside dropdown for these submenu controls should be appended in sub menu too. Visual and screen reader user experience should be consistent through out the app .

**Test Environment:**
App: Windows Terminal Preview
Feature: Command Palette
Screen Reader: Narrator
OS: Win 10 2004 Build (19041.508)

**Repro Steps:**
1.Open Windows terminal Preview app
2.Open Command Palette using shortcut (Ctrl+shift+p)
3.Search for New tab command in the search field
4.Navigate to New Tab list item having sub menu and activate it .
5.verify whether the shortcuts are appended with the list items in New tab are not.

**Actual:**
Keyboard shortcuts are not included in the sub menu of New Tab list item.

**Expected:**
Keyboard shortcuts should be included in the sub menu of New Tab list item like they are included in the drop down of windows terminal for the  same list items.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B64DD7C5C-AA62-4D53-886E-BCAEE5341C01%7D&file=Guideline%20101%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties_v3.docx&action=default&mobileredirect=true

**Comment by zadjii-msft** [comment]
This is by design, the sub-commands of the "New tab..." command only have shortcut text if the user has a keybinding specifically for that action. Note that there's a difference between the actions specified by

```json
{ "command": { "action": "newTab", "index": 0 }, "keys": "ctrl+shift+1" },
```

and
 
```json
{ "command": { "action": "newTab", "profile": "profile0" }, "keys": "ctrl+shift+1" }
```

Even if "profile0" is the first profile the user has. If the user has the second action in their settings, the shortcut text _will_ appear for the nested menu item:

![image](https://user-images.githubusercontent.com/18356694/95858313-8668db00-0d22-11eb-84db-a0f2a8f91337.png)

**Comment by zadjii-msft** [comment]
@Chandresh1007 @Priyanka-Chauhan123 @DasaradhaKamjula Since this is by design, can we close this issue?

**Comment by Priyanka-Chauhan123** [comment]
Agree with the above comments as the shortcuts are appearing for the nested menu items and closing this bug.
